.github/workflows/CI.yml

```
@@ -28,9 +31,12 @@ jobs:
           ${{ runner.OS }}-
     - name: Install dependencies
       run: npm ci
+    - name: Install dependencies for integration test
+      if: ${{ matrix.integration-deps != '' }}
+      run: npm install ${{ matrix.integration-deps }}
     - name: Build
       env:
-        COVERAGE: 1
+        COVERAGE: ${{ matrix.integration-deps != '' }}
```

Shouldn't coverage be
```suggestion
        COVERAGE: ${{ matrix.integration-deps == '' }}
```
so we include the bpmn-js@9 only tests?

---

Right! 😅